suite-native/module-trading/src/components/sell/SellPreview/__tests__/SellPreviewScreenHeader.comp.test.tsx

```diff
@@ -0,0 +1,14 @@
+import { renderWithStoreProviderAsync } from '@suite-native/test-utils';
+
+import { SellPreviewScreenHeader } from '../SellPreviewScreenHeader';
+
+describe('SellPreviewScreenHeader', () => {
+    const renderSellPreviewScreenHeader = () =>
+        renderWithStoreProviderAsync(<SellPreviewScreenHeader />);
```

Comment:
Personally I find this a bit redundant, it needed I would test this along `TradingSellPreviewScreen`